merkledag/README.md

```diff
@@ -0,0 +1,144 @@
+# The merkledag
+
+
+Authors: [Juan Benet](github.com/jbenet)
+
+Reviewers:
+
+- [Jeromy Johnson](github.com/whyrusleeping)
+
+* * *
+
+This is the [Spec](../) for the IPFS Merkledag.
+
+## Definitions
+
+- `hash` - throughout this document, the word `hash` refers specifically to cryptographic hash functions, such as sha3.
+- `dag` - directed acyclic graph
+- `merkle-link` - a link (graph edge) between two objects, which is (a) represented by the hash of the target object, and (b) embedded in the source object. merkle-links construct graphs (dags) whose links are content-addressed, and authenticated.
```

Comment:
content addressed data structures can be used to make authenticated data structures. 

authenticated data structures are a crypto idea. it's a data structure which can be manipulated and verified by agents who do not trust each other.